clarity-ext-scripts/clarity_ext_scripts/covid/parse_pcr_execution.py

```diff
@@ -2,33 +2,75 @@
 
 import pandas as pd
 from datetime import datetime
+from clarity_ext.domain.validation import UsageError
 
-CT_HEADER = u"Cт"
+CT_HEADER_OLD = u"Cт"
```

Comment:
I guess this can just be removed, since it is not used?

Reply:
Ok, I remove it. My thought was to save unicode T since it's awkward to get, when we want to parse the other instrument file.